cypress/integration/smokeTests/sharedTests/auth-tests.ts

```diff
@@ -251,6 +257,25 @@ function testWorkflow(registry: string, repo: string, name: string) {
       cy.contains('button', 'Restub').click();
       cy.contains('button', 'Publish').should('be.disabled');
     });
+    it('hide and un-hide a version', () => {
+      function toggleHidden() {
```

Comment:
Nitpicky: would be great if both versions of `toggleHidden` were defined at the same lexical level.

Reply:
Done.